internal/grpc/interceptors/auth/auth.go

```diff
@@ -68,7 +59,7 @@ func parseConfig(m map[string]interface{}) (*config, error) {
 
 func skip(url string, skipped []string) bool {
 	for _, s := range skipped {
-		if strings.HasPrefix(s, url) {
```

Comment:
We could DRY this skip function and export the `skip` method from the http auth middleware

https://github.com/cs3org/reva/blob/master/internal/http/interceptors/auth/auth.go#L79-L88

or at least write a test for this one to be on the safe side

https://github.com/cs3org/reva/blob/master/internal/http/interceptors/auth/auth_test.go

I totally forgot to update the interceptors when updated the skip on the HTTP middlewares

Reply:
Complete agree :)

Reply:
I've moved the skip logic to a utils package (yes, I know but I could'nt find a better name) :)
@refs Let me know what you think

Reply:
`utils` although against [package naming conventions](https://blog.golang.org/package-names) should be good enough for us. We can always refactor this in the future 🚀

Reply:
@refs  I couldn't find a better name to throw it, any naming better than this will do :)